sync/class.jetpack-sync-defaults.php

```diff
@@ -335,6 +335,17 @@ public static function get_post_meta_whitelist() {
 		'hc_foreign_user_id'
 	);
 
+	public static function get_comment_meta_whitelist() {
+		/**
+		 * Filter the list of comment meta data that are manageable via the JSON API.
+		 *
+		 * @module sync
```

Comment:
@gravityrail We'll need the `@since 5.7.0` line here. Do you think you could add it in a quick PR?

Thanks!

Reply:
Done: https://github.com/Automattic/jetpack/pull/8349